.github/workflows/shiftleft-terraform.yaml

```diff
@@ -0,0 +1,26 @@
+---
+# This workflow integrates ShiftLeft NG SAST with GitHub
+# Visit https://docs.shiftleft.io for help
+name: shiftleft-terraform
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  shiftleft-terraform:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+    - run: echo "REPO_NAME=${GITHUB_REPOSITORY#*/}" >> $GITHUB_ENV
+    - name: Download ShiftLeft CLI
+      run: |
+        curl https://cdn.shiftleft.io/download/sl > ${GITHUB_WORKSPACE}/sl && chmod a+rx ${GITHUB_WORKSPACE}/sl
+    - name: Extract branch name
+      shell: bash
+      run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+      id: extract_branch
+    - name: NextGen Static Analysis
+      run: ${GITHUB_WORKSPACE}/sl analyze --strict --wait --app ${{ env.REPO_NAME}} --tag branch=${{ github.head_ref || steps.extract_branch.outputs.branch }} --terraform $(pwd)
+      env:
+        SHIFTLEFT_ACCESS_TOKEN: ${{ secrets.SHIFTLEFT_ACCESS_TOKEN }}
```

Comment:
Have we tried running this yet? App should show up in Shiftleft dashboard.

Reply:
I see a green checkmark in workflows below. Looks to be running for me.

Reply:
Additionally I have no access to shiftleft

Reply:
<img width="1139" alt="Screen Shot 2565-02-02 at 10 18 32" src="https://user-images.githubusercontent.com/2181540/152088011-5763019e-ef96-4c92-a934-0e6ee9100375.png">
looks like its there